### DISCUSSION TITLE
Is there a similar mechanism to the Sphinx directive deprecated?

### DISCUSSION THREAD

**Discussion by Midnighter** [mkdocs/mkdocs#2390] [Q&A] [answered]
In Sphinx there are some [handy directives](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated) for annotating functions or methods that changed or became deprecated in a specific version. Is there something similar that I can use in mkdocs?

**Comment by waylan** [comment, marked as answer]
Markdown does not have directives. That said, there may be [third party extensions][1] to Markdown which provide similar features. You can configure extensions using the [markdown_extensions][2] config setting.

You might also want to check out some of the MkDocs plugins in the [API documentation][3] section.

[1]: https://github.com/Python-Markdown/markdown/wiki/Third-Party-Extensions
[2]: https://www.mkdocs.org/user-guide/configuration/#markdown_extensions
[3]: https://github.com/mkdocs/mkdocs/wiki/MkDocs-Plugins#api-documentation-building
